### ISSUE TITLE
Eliminate sprite

### ISSUE THREAD

**Issue by mikeselander** [greenpeace/planet4-presentation-layer#18]
Sprite are not particularly useful tools in a modern build on servers that can support concurrent downloads - we should use SVGs or fa icons.

**Comment by 27Bslash6** [comment]
Agreed, will add that I'm advocating installation of Google Pagespeed module in infrastructure which will likely further enhance this approach by converting small images to data-uris, further reducing FOUC / re-layout.

Currently in use at https://docker.p4.greenpeace.org/

**Comment by mikeselander** [comment]
This should be considered an optimization and really isn't required for the MVP - punting to Post-MVP.
